array-manipulation-3/src/equal.js

```diff
@@ -0,0 +1,10 @@
+/* exported equal */
+function equal(first, second) {
+  const firstString = JSON.stringify(first);
+  const secondString = JSON.stringify(second);
```

Comment:
This is a risky approach. Instead, check their lengths with a guard, then loop through the first and compare the second at each index.